src/idl_gen_json_schema.cpp

```diff
@@ -174,9 +174,16 @@ class JsonSchemaGenerator : public BaseGenerator {
                       ",\n                \"maxItems\": " +
                       NumToString(property->value.type.fixed_length);
         }
+        std::string deprecatedInfo = "";
```

Comment:
please use `snake_case` rather than `camelCase` here and elsewhere

Reply:
should I rename other camelCase variables around but not related to my changes? better in separate commit?

Reply:
I've changed only variable introduced in my commit

Reply:
would have been fine to fix the variables you're using also, but no big deal either way.